pkg/verify/verify.go

```diff
@@ -245,6 +184,7 @@ func (v *releaseVerifier) Verify(ctx context.Context, releaseDigest string) erro
 				continue
 			}
 			delete(remaining, k)
```

Comment:
There's some contention between efficiently verifying a digest, where you only need one sig for each keyring, and shipping sigs around, where you may want too keep walking to find the "best" valid sig for each keyring.  Best might be "all valid sigs" or "latest expiration date" or "strongest cypher" or whatever.  Do we need a mode toggle to switch between efficient verification and interesting-sig collection?

Reply:
If we get into that model, we could do that.  But I think that's cart before horse, and if we need it we would have the cache function potentially manage that.